fix for "omitting directory" error on installing of 'inc/common' during 'make install'
author: Kenneth Hoste (HPC-UGent)
--- ncbi_cxx--18_0_0/src/build-system/Makefile.in.top.orig	2017-06-29 14:51:43.855712613 +0200
+++ ncbi_cxx--18_0_0/src/build-system/Makefile.in.top	2017-06-29 14:54:47.876416764 +0200
@@ -51,7 +51,8 @@
 	    done
 	cd $(includedir0) && find * -name CVS -prune -o -print |\
             cpio -pd $(pincludedir)
-	$(INSTALL) -m 644 $(incdir)/* $(pincludedir)
+	$(INSTALL) -m 644 $(incdir)/*.h $(pincludedir)
+	$(INSTALL) -m 644 $(incdir)/common/* $(pincludedir)/common
 ## set up appropriate build and status directories somewhere under $(libdir)?
 
 install-gbench:
